crates/extensions/tedge_health_ext/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+mod actor;
+use actor::HealthManagerMessageBox;
+use actor::TedgeHealthMonitorActor;
+use tedge_actors::futures::channel::mpsc;
+use tedge_actors::Builder;
+use tedge_actors::CombinedReceiver;
+use tedge_actors::DynSender;
+use tedge_actors::LinkError;
+use tedge_actors::MessageSink;
+use tedge_actors::MessageSource;
+use tedge_actors::NoConfig;
+use tedge_actors::RuntimeRequest;
+use tedge_actors::RuntimeRequestSink;
+use tedge_mqtt_ext::MqttActorBuilder;
+use tedge_mqtt_ext::MqttMessage;
+
+type HealthInputMessage = MqttMessage;
+type HealthOutputMessage = MqttMessage;
+
+pub struct HealthMonitorBuilder {
+    input_receiver: CombinedReceiver<HealthInputMessage>,
+    events_sender: mpsc::Sender<HealthInputMessage>,
+    mqtt_publisher: Option<DynSender<HealthOutputMessage>>,
+    signal_sender: mpsc::Sender<RuntimeRequest>,
+}
+
+impl HealthMonitorBuilder {
+    pub fn new() -> Self {
+        let (events_sender, events_receiver) = mpsc::channel(10);
+        let (signal_sender, signal_receiver) = mpsc::channel(10);
+        let input_receiver = CombinedReceiver::new(events_receiver, signal_receiver);
+
+        Self {
+            input_receiver,
+            events_sender,
+            mqtt_publisher: None,
+            signal_sender,
+        }
+    }
+
+    /// Connect this config manager instance to some mqtt connection provider
+    pub fn with_mqtt_connection(&mut self, mqtt: &mut MqttActorBuilder) -> Result<(), LinkError> {
+        let subscriptions = vec!["tedge/health-check", "tedge/health-check/+"]
```

Comment:
```suggestion
        let subscriptions = vec!["tedge/health-check", "tedge/health-check/c8y-device-management"]
```
And as suggested earlier, read that daemon name from a struct field so that it can be changed via configurations in future.